core/src/main/java/com/hcl/ecomm/core/services/ProductService.java

```diff
@@ -13,11 +15,11 @@
 
 	public String getSearchCriteriaValue();
 
-	public JsonArray getAllProductDetails();
+	public JSONArray getAllProductDetails() throws JSONException;
```

Comment:
@Subrat-Dash-416  Why this class changed?

Reply:
@rakeshkumar-verma  This change was made to resolve the google Gson error which we were facing .Discussed the same with you ,to follow the json.org approach.